Library/Homebrew/cmd/info.rb

```diff
@@ -45,6 +45,8 @@ class Info < AbstractCommand
         switch "--github",
                description: "Open the GitHub source page for <formula> and <cask> in a browser. " \
                             "To view the history locally: `brew log -p` <formula> or <cask>"
+        switch "--github-manifest",
+               description: "Fetch GitHub package manifest for extra information when <formula> is not installed."
```

Comment:
Would be `--github-packages-manifest` be better to match `--github-packages-downloads`? Though longer name is less ideal for users
```suggestion
        switch "--github-packages-manifest",
               description: "Fetch GitHub Packages manifest for extra information when <formula> is not installed."
```